pkg/doozer/prepare/dockerfile.go

```diff
@@ -93,6 +94,10 @@ func NewDockerFile(dockerSpec global.DockerSpec, auroraVersion runtime.AuroraVer
 		env := createEnv(auroraVersion, dockerSpec.PushExtraTags, imageBuildTime)
 
 		dockerFileTemplate := dockerFileTemplateBody
+		if strings.Contains(baseImage.GetCompleteDockerTagName(), "rumple") {
+			dockerFileTemplate += "RUN $HOME/application/bin/install-deps.sh\n"
```

Comment:
I dag er det som er i leveransepakken den komplette applikasjonen. Det er et designvalg som ligger langt tilbake (før min tid), men det er noe vi har fulgt hittil.

Det å kjøre installering av avhengigheter i byggeprosessen, bryter med den.

Er det noen grunn til at leveransepakken ikke kan bygges inn med alle avhengigheter?

Reply:
Nei, og det er planen for den endelige løsningen. Men det krever endring i pipeline script/evt. eget byggeimage for python som er en vesentlig større jobb. Dette er tenkt å løse behov for å pakke python applikasjoner i images på en måte som fungerer med dagens oppsett, og deretter jobbe med en mer komplett løsning som tar høyde for fremtidig architect/pipelines.